tuna/miopen/subcmd/import_configs.py

```diff
@@ -115,18 +115,7 @@ def insert_config(driver: DriverBase, counts: dict, dbt: MIOpenDBTables,
       except IntegrityError as err:
         logger.warning("Err occurred: %s", err)
         session.rollback()
-    else:
-      try:
-        if args.mark_recurrent or args.tag:
```

Comment:
this is the only place we are using the recurrent argument. If this goes, then we should remove the recurrent argument and col in the DB as well. I dont think we have used it in a long time

Reply:
check line 130